frontend/__tests__/components/FilterCategory.test.tsx

```diff
@@ -0,0 +1,17 @@
+import { render, screen } from '@testing-library/react'
+import '@testing-library/jest-dom'
+
+import FilterCategory from '../../src/components/FilterCategory'
+
+describe(FilterCategory, () => {
+  it('displays a filter button for each option', () => {
+    const OPTIONS = ['Option 1', 'Option 2', 'Option 3']
```

Comment:
Non-blocking nit: conventionally we use all-capitalized variable names for things that are globally-scoped

Reply:
this is a good point. i was feeling a bit weird about the all-caps and couldn't pin point why.